services/web/server/src/simcore_service_webserver/groups_api.py

```diff
@@ -101,7 +101,9 @@ async def _get_user_group(conn: SAConnection, user_id: int, gid: int) -> RowProx
 async def _get_user_from_email(app: web.Application, email: str) -> RowProxy:
     engine = app[APP_DB_ENGINE_KEY]
     async with engine.acquire() as conn:
-        result = await conn.execute(sa.select([users]).where(users.c.email == email))
+        result = await conn.execute(
+            sa.select([users]).where(users.c.email == email.lower())
+        )
```

Comment:
Same comment as above, please check the source(s) and apply lower case at the closes point of entry in the application.